docs/imagemanifestv2.md

```diff
@@ -0,0 +1,113 @@
+# Image Manifest V2 
+
+This document outlines the format of of the V2 image manifest.  Image manifests describe the various constituents of a docker image.  Image manifests can be serialized to JSON format.
+
```

Comment:
Please reference https://github.com/docker/docker/issues/8093 and https://github.com/docker/docker/issues/9015 as part of the origins of this format.